dockerfiles/centos/Dockerfile

```diff
@@ -36,6 +36,11 @@ WORKDIR /opt/app
 
 COPY . .
 
+RUN yum install -y git
+RUN git init
+RUN git config --global url."https://github.com".insteadOf git@github.com:
+RUN git config --global url."https://".insteadOf git://
```

Comment:
@tjduigna Are you sure this is needed? `npm install` works fine without this setup for me:

```
➜  madejs git:(dev) ✗ cat ~/.gitconfig                                                                                         
[push]
	default = current
[user]
	name = Timur Bazhirov
	email = timur@exabyte.io
[core]
	editor = vim
[diff]
    submodule = log
➜  madejs git:(dev) ✗ npm install git+https://github.com:exabyte-io/periodic-table#8e2cd63c411f35fede2a466252871d81df20754c

> @exabyte-io/periodic-table.js@2021.12.21-6 postinstall /Users/timur/Code/madejs/node_modules/@exabyte-io/periodic-table.js
> npm run transpile


> @exabyte-io/periodic-table.js@2021.12.21-6 transpile /Users/timur/Code/madejs/node_modules/@exabyte-io/periodic-table.js
> mkdir -p lib; babel src --out-dir lib; node lib/js/generate.js

Successfully compiled 2 files with Babel (904ms).
npm WARN @babel/eslint-parser@7.16.3 requires a peer of @babel/core@>=7.11.0 but none is installed. You must install peer dependencies yourself.
npm WARN @babel/eslint-parser@7.16.3 requires a peer of eslint@^7.5.0 || ^8.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.16.7 requires a peer of @babel/core@^7.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN @babel/plugin-proposal-class-static-block@7.16.7 requires a peer of @babel/core@^7.12.0 but none is installed. You must install peer dependencies yourself.
npm WARN @babel/helper-define-polyfill-provider@0.3.1 requires a peer of @babel/core@^7.4.0-0 but none is installed. You must install peer dependencies yourself.

+ @exabyte-io/periodic-table.js@2021.12.21-6
updated 1 package and audited 512 packages in 4.525s

9 packages are looking for funding
  run `npm fund` for details

found 8 vulnerabilities (1 low, 1 moderate, 5 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
➜  madejs git:(dev) ✗ 
```

Reply:
Did you run it in docker? I experienced the same problem for the test container because the `tests/package-lock.json` has references to the following packages
```
"cucumber": "github:xolvio/cucumber-js#cf953cb5b5de30dbcc279f59e4ebff3aa040071c",
"glob": "github:lucetius/node-glob#51c7ca6e69bfbd17db5f1ea710e3f2a7a457d9ce",
```
Which by default will ask for SSH credentials when pulling the repositories. Since it was the same error for the centos container, and the same configuration fixes it, I assume it's necessary.